map_server/CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ if(BUILD_TESTING)
   )
 endif()
 
+ament_export_libraries(${PROJECT_NAME})
```

Comment:
thanks, I copied it from elsewhere and it 'just worked'. Actually, even if I put `ament_export_libraries("garbage")` it fixes my issue... I think this is because the [checking is deferred to the `find_library` calls](https://github.com/ament/ament_cmake/blob/3610daaaa978e3b79f08bafa5bdf6a7c3300b201/ament_cmake_export_libraries/cmake/ament_export_libraries.cmake#L121) later on, but I am mentioning it in case that seems fishy to you.